CHANGELOG.md

```diff
@@ -4,6 +4,16 @@ All notable changes to this project will be documented in this file.
 Each new release typically also includes the latest modulesync defaults.
 These should not affect the functionality of the module.
 
+## [v4.0.1](https://github.com/voxpupuli/puppet-systemd/tree/v4.0.1) (2023-01-30)
+
+[Full Changelog](https://github.com/voxpupuli/puppet-systemd/compare/v4.0.0...v4.0.1)
+
+**Fixed bugs:**
+
+- Udev fact exceeds the number of facts limit [\#314](https://github.com/voxpupuli/puppet-systemd/issues/314)
```

Comment:
I added skip-changelog for the bug, because we already have the related pull request in the changelog. please regenerate.

Reply:
Copy. I've never been sure what to do in that scenario. I've generally tried to make sure the issue + pr have the same tag.

Reply:
Repushed.